File: src/main/java/no/nav/service/pensjon/regler/to/BeregnPoengtallBatchRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Vector;

import no.nav.domain.pensjon.regler.grunnlag.PersonOpptjeningsgrunnlag;

public class BeregnPoengtallBatchRequest extends ServiceRequest{

	private Vector<PersonOpptjeningsgrunnlag> personOpptjeningsgrunnlagListe = new Vector<PersonOpptjeningsgrunnlag>();

}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnPoengtallBatchRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.grunnlag.PersonOpptjeningsgrunnlag


class BeregnPoengtallBatchRequest : ServiceRequest() {
    var personOpptjeningsgrunnlagListe: MutableList<PersonOpptjeningsgrunnlag?> = mutableListOf()
}